.env.vagrant

```diff
@@ -34,6 +32,7 @@ OAUTH2_CLIENT_ID=
 OAUTH2_CLIENT_SECRET=
 
 COMPANY_AUTH_VERIFICATION_FEATURE_ENABLED=1
+ILLNESS_REASON_FEATURE_ENABLED=1
```

Comment:
I love the fact you made it consistent, although remember changing it in two more config repositories.